Bottom: 8a85b5c770c9e3442093af4bb1b69eb4ce7e9a6f
Top:    e65d88ebb2dc88c7ec007579c4536b00468e1640
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:09:08 -0500

Refresh of codebase-generator

---
diff --git a/roles/buildbot-master/files/master.cfg b/roles/buildbot-master/files/master.cfg
index c14bd7dd..5a366551 100644
--- a/roles/buildbot-master/files/master.cfg
+++ b/roles/buildbot-master/files/master.cfg
@@ -172,7 +172,7 @@ def codebaseGenerator(chdict):
     """
 
     giturl = giturlparse(chdict['repository'])
-    return giturl.owner
+    return '/'.join((giturl.owner, giturl.repo))
 
     # path = urlparse(chdict['revlink']).path.lstrip('/')
